### ISSUE TITLE
No trailing newline breaks parser

### ISSUE THREAD

**Issue by jphastings** [cprice404/ruby-hocon#19]
If there is no trailing newline in a conf file (with v0.0.6) then the parser breaks:

```
gems/hocon-0.0.6/lib/hocon/impl/parser.rb:482:in `parse_object': uninitialized constant Hocon::Impl::Tokens::END (NameError)
from gems/hocon-0.0.6/lib/hocon/impl/parser.rb:581:in `parse'
from gems/hocon-0.0.6/lib/hocon/impl/parser.rb:973:in `parse'
from gems/hocon-0.0.6/lib/hocon/impl/parseable.rb:186:in `raw_parse_value_from_io'
from gems/hocon-0.0.6/lib/hocon/impl/parseable.rb:180:in `block in raw_parse_value'
from gems/hocon-0.0.6/lib/hocon/impl/parseable.rb:34:in `block in open'
from gems/hocon-0.0.6/lib/hocon/impl/parseable.rb:33:in `open'
from gems/hocon-0.0.6/lib/hocon/impl/parseable.rb:33:in `open'
from gems/hocon-0.0.6/lib/hocon/impl/parseable.rb:179:in `raw_parse_value'
from gems/hocon-0.0.6/lib/hocon/impl/parseable.rb:163:in `parse_value_from_origin'
from gems/hocon-0.0.6/lib/hocon/impl/parseable.rb:116:in `parse_value'
from gems/hocon-0.0.6/lib/hocon/impl/parseable.rb:99:in `parse'
from gems/hocon-0.0.6/lib/hocon/config_factory.rb:7:in `parse_file'
```

**Comment by fpringvaldsen** [comment]
@jphastings Thanks for the heads up! I just released v0.0.7, which contains a fix for this issue. I'm going to close it out for now, but please reopen if you see this crop up again.

**Comment by fpringvaldsen** [comment]
Fixed in https://github.com/cprice404/ruby-hocon/pull/17
